src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,24 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int sizeReturnedString = 5;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
I will try to be more attentive